### PR TITLE
[FIX] *: correct typso and English errrors

### PR DESCRIPTION
**Pull request by mart-e**

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
I like to correct typso .

File: addons/barcodes_gs1_nomenclature/models/barcode_rule.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def _default_encoding(self):
         ('measure', 'Measure'),
         ('identifier', 'Numeric Identifier'),
         ('alpha', 'Alpha-Numeric Name'),
-    ], string="GS1 Content Type", help="The GS1 content type defines to what this rule data will be handle:\
+    ], string="GS1 Content Type",
+        help="he GS1 content type defines what kind of data the rule will process the barcode as:\
```
Comment (ticodoo) [inline addons/barcodes_gs1_nomenclature/models/barcode_rule.py:45]: "The" lost a "T"?

File: addons/barcodes_gs1_nomenclature/i18n/barcodes_gs1_nomenclature.pot [hunk 2]
```diff
@@ -160,16 +160,6 @@ msgstr ""
 msgid "The FNC1 Seperator Alternative is not a valid Regex: "
```
Comment (ticodoo) [inline addons/barcodes_gs1_nomenclature/i18n/barcodes_gs1_nomenclature.pot:160]: "Seperator" should be "Separator", but I was too lazy to open PR just to fix the typo where it shows up. May as well fix it as part of this PR?

**Comment by mart-e** [comment]
@robodoo r+

**Comment by mart-e** [comment]
@robodoo r+

**Comment by mart-e** [comment]
robodoo r+

**Pull request closed without merging**